lab-jonathan/lib/errors.js

```diff
@@ -0,0 +1,9 @@
+let createError = require('http-errors');
```

Comment:
I didn't see this before my last comment, but you still need to import this into your server file to handle errors there via middleware.